### PR TITLE
Starting to turn on set -e for test scripts

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #169 

I turned on set -e in a bunch of scripts, but when I got to test_fms/drifters I encountered two failing tests, so I stopped there until we can sort them out.

The two tests are test_drifters.sh and test_drifters_input.sh. They cause the test to fail. Apparently the "skip" command is the problem. Can we find another way to handle skipping tests? I couldn't figure out any way to comment out lines in the bats file such that a success would be returned.

So basically you are running bats but not checking the return code. The bats command can fail in the sh, and success will still be returned.

This PR includes #200 and #201.

### CODE REVIEW

File: test_fms/drifters/Makefile.am [hunk 1]
```diff
@@ -31,9 +31,9 @@ test_quicksort_SOURCES = test_quicksort.F90
 test_drifters_SOURCES = test_drifters.F90
 
 # Run the test programs.
-TESTS = test_drifters_io.sh test_cloud_interpolator.sh \
-test_drifters_input.sh test_drifters_comm.sh test_drifters_core.sh \
-test_quicksort.sh test_drifters.sh
+TESTS = test_drifters_io.sh test_cloud_interpolator.sh		\
+test_drifters_comm.sh test_drifters_core.sh test_quicksort.sh
+#test_drifters.sh test_drifters_input.sh
```
Comment (underwoo) [inline test_fms/drifters/Makefile.am]: Why are these files commented out?

Comment (edwardhartnett) [reply test_fms/drifters/Makefile.am]: They are not, I just added one and hit the auto re-indent in emacs.

**Comment by edwardhartnett** [comment]
I have made the requested changes here.

**Comment by edwardhartnett** [comment]
This PR can be skipped if we are going to see #228 merged soon. #228 handles the set -e issue everywhere.
